src/stylesheets/main.css

```diff
@@ -202,6 +202,60 @@ footer a:hover {
   text-decoration: underline;
 }
 
+.press-release img {
+  display: block;
+  max-width: 100%;
+}
+
+.press-release figcaption p {
+  font-size: 0.875em;
+  font-style: italic;
+  text-align: center;
+}
+
+.press-release sup a {
+  font-size: 1em;
+  margin: 0 0.15em;
+}
+
+.footnotes ol {
+  padding-left: 1rem;
+}
+
+.footnotes ol::before {
+  content: "Footnotes";

```

Comment:
I'm not concerned about this from an accessibility perspective because the `<div>` that Kramdown generates has a `role="doc-endnotes"`.